packages/graphback/src/buildGraphbackAPI.ts

```diff
@@ -0,0 +1,106 @@
+import { GraphQLSchema } from 'graphql';
+import { GraphbackPlugin, GraphbackPluginEngine, GraphbackCRUDGeneratorConfig, printSchemaWithDirectives, ModelDefinition } from '@graphback/core';
+import { SchemaCRUDPlugin } from '@graphback/codegen-schema';
+import { LayeredRuntimeResolverCreator, GraphbackCRUDService, DataProviderCreator, createCRUDService } from '@graphback/runtime';
+import { PubSub } from 'graphql-subscriptions';
+
+export interface GraphbackAPIConfig {
+  /**
+   * Function which creates a default CRUD Service for every data model
+   */
+  serviceCreator?: Function
```

Comment:
Hmm.. the only drawback of this is to not being able to define format of the return type from function. I think we can use typescript argument array format to build top level function but then return strongly typed creator one